tcms/testcases/templates/testcases/search.html

```diff
@@ -133,7 +133,7 @@
         <table class="table table-striped table-bordered table-hover" id="resultsTable">
             <thead>
                 <tr>
-                    <th class="js-select-checkbox"></th>
+                    <th class="js-select-checkbox"><input id="check_all" type="checkbox"></th>
```

Comment:
we don't have a convention for that, but I think IDs are supposed to be kebap-case - `check-all`

Reply:
doesn't really matter. No convention and the naming style is all over the place.